File: apzkr-pzpi-21-5-vyshniak-viktoriia/Task1-Server/BLL/Infrastructure/Models/Pet/PetListItem.cs
using BLL.Infrastructure.Models.Arduino;
using Common.Enums;

namespace BLL.Infrastructure.Models.Pet;
public class PetListItem
{
    public Guid PetId { get; set; }

    public string NickName { get; set; }

    public PetType PetType { get; set; }

    public string OwnerLogin { get; set; }

    public ArduinoSettingsModel ArduinoSettings { get; set; }
}

File: apzkr-pzpi-21-5-vyshniak-viktoriia/Task3-WebClient/src/app/core/models/pet/pet-list-item.ts
import { Guid } from "guid-typescript";
import { PetType } from "../../enums/pet-type";
import { ArduinoSettingsModel } from "./arduino-settings-model";

export interface PetListItem {
	petId: Guid;
	nickName: string;
	petType: PetType;
	arduinoSettings: ArduinoSettingsModel,
	ownerLogin: string,
}
